Mage.Sets/src/mage/cards/o/OfficiousInterrogation.java

```diff
@@ -0,0 +1,123 @@
+package mage.cards.o;
+
+import java.util.UUID;
+
+import mage.abilities.Ability;
+import mage.abilities.SpellAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.cost.CostModificationEffectImpl;
+import mage.abilities.effects.keyword.InvestigateEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.CostModificationType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.filter.StaticFilters;
+import mage.game.Game;
+import mage.target.Target;
+import mage.target.TargetPlayer;
+
+/**
+ *
+ * @author DominionSpy
+ */
+public final class OfficiousInterrogation extends CardImpl {
+
+    public OfficiousInterrogation(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{W}{U}");
+
+        // This spell costs {W}{U} more to cast for each target beyond the first.
+        this.addAbility(new SimpleStaticAbility(Zone.STACK, new OfficiousInterrogationCostIncreasingEffect())
+                .setRuleAtTheTop(true));
+
+        // Choose any number of target players. Investigate X times, where X is the total number of creatures those players control.
```

Comment:
Must be research: is it possible to put card hint with dynamic value like “Number of creatures: 0”. Can be useful on stack.